### PR TITLE
Remove tooltip interactive map mobile

### PR DESCRIPTION
**Pull request by MarianneLa**

### CODE REVIEW

File: src/modules/Map/InteractiveMap/InteractiveMap.js [hunk 1]
```diff
@@ -367,10 +369,11 @@ export class InteractiveMap extends React.Component {
     element,
     className,
   }) => {
-    const { history } = this.props;
+    const { history, isMobileSized } = this.props;
     const { map, isPointerOverMap } = this;
 
     if (isPointerOverMap === false) return;
+    if (isMobileSized) return;
```
Comment (dtrucs) [inline src/modules/Map/InteractiveMap/InteractiveMap.js]: The `isMobileSized` boolean only defines the size of the screen. I'm not sure it's the best way to detect if the client has a touchscreen display or not.

In this case, i would remove the HOC and test the following condition instead: 
`if(!global.matchMedia("(hover: hover)").matches) return`

File: src/modules/Map/InteractiveMap/InteractiveMap.test.js [hunk 2]
```diff
@@ -865,6 +865,22 @@ describe('Interactions', () => {
...
+    });
+    instance.setState = () => null;
+    instance.componentDidMount();
+    // instance.displayTooltip({
+    //   layerId: 'foo',
+    //   features: [{ properties: {} }],
+    //   event: { lngLat: { lng: 3, lat: 4 } },
+    //   template: 'bar',
+    // });
+
```
Comment (dtrucs) [inline src/modules/Map/InteractiveMap/InteractiveMap.test.js]: Maybe this test is irrelevant

File: src/modules/Map/InteractiveMap/InteractiveMap.js [hunk 3]
```diff
@@ -661,4 +663,4 @@ export class InteractiveMap extends React.Component {
   }
 }
 
-export default connectState('initialState')(InteractiveMap);
+export default connectState('initialState')((InteractiveMap));
```
Comment (mabhub) [inline src/modules/Map/InteractiveMap/InteractiveMap.js]: There's an extraneous pair of parens.

File: src/modules/Map/InteractiveMap/InteractiveMap.js [hunk 4]
```diff
@@ -133,6 +132,7 @@ export class InteractiveMap extends React.Component {
     })),
     onInit: PropTypes.func,
     onStyleChange: PropTypes.func,
+    isMobileSized: PropTypes.bool,
```
Comment (dtrucs) [inline src/modules/Map/InteractiveMap/InteractiveMap.js]: This propTypes is defined but never called after

File: src/modules/Map/InteractiveMap/InteractiveMap.js [hunk 5]
```diff
@@ -141,6 +141,7 @@ export class InteractiveMap extends React.Component {
     legends: [],
     onInit () {},
     onStyleChange () {},
+    isMobileSized: false,
```
Comment (dtrucs) [inline src/modules/Map/InteractiveMap/InteractiveMap.js]: This defaultProps is setted but never called after

File: locales/fr/translation.json [hunk 6]
```diff
@@ -144,8 +145,8 @@
         }
       },
       "too_many_results": "Votre recherche contient trop de résultats, veuillez affiner votre requête ou zoomer sur une zone plus restreinte.",
-      "fold": "Relier",
+      "too_many_results_plural": "",
```
Comment (dtrucs) [inline locales/fr/translation.json]: Maybe this line is useless?

Comment (MarianneLa) [reply locales/fr/translation.json]: done

File: src/modules/Map/InteractiveMap/InteractiveMap.js (hunk 5 above)
Comment (MarianneLa) [reply src/modules/Map/InteractiveMap/InteractiveMap.js]: done

File: src/modules/Map/InteractiveMap/InteractiveMap.js (hunk 4 above)
Comment (MarianneLa) [reply src/modules/Map/InteractiveMap/InteractiveMap.js]: done

File: src/modules/Map/InteractiveMap/InteractiveMap.js (hunk 3 above)
Comment (MarianneLa) [reply src/modules/Map/InteractiveMap/InteractiveMap.js]: done

**Pull request merged**